aries_vcx/src/handlers/proof_presentation/prover.rs

```diff
@@ -191,23 +197,28 @@ impl Prover {
         Ok(())
     }
 
+    // TODO: Can we delete this (please)?
```

Comment:
Ultimately having option to decline request can be quite common requirement. We don't have to necessary address it in this PR, but splitting the function into 2 functions 
```
decline_presentation_request(..., reason)
propose_presentation_request(...,proposal)
```
could make the implementation a lot more readible.

Reply:
Obviously, I meant delete and replace ;)